docs/overview.md

```diff
@@ -0,0 +1,72 @@
+# Overview
```

Comment:
I don't see this file linked to from `mkdocs.yml`; is it orphaned?

Also this duplicates a lot of what's in `index.md`.

Reply:
Tracking these - I just resolved and merged. thanks